static/src/js/components/SpatialDisplay/SpatialSelectionDropdown.jsx

```diff
@@ -21,16 +21,15 @@ import EDSCIcon from '../EDSCIcon/EDSCIcon'
 
 import './SpatialSelectionDropdown.scss'
 
-export class SpatialSelectionDropdown extends PureComponent {
-  constructor(props) {
-    super(props)
-    this.onItemClick = this.onItemClick.bind(this)
-  }
+const SpatialSelectionDropdown = (props) => {
```

Comment:
I wonder if this should be memoized with `React.memo`, since it was a pure component previously